### DISCUSSION TITLE
titlebar customization with gtk backend

### DISCUSSION THREAD

**Discussion by ivanjx** [unoplatform/uno#10361] [Q&A]
is titlebar customization available with gtk backend? i cant seem to even set window title and the log says that it is not supported. i also need to hide the default title bar and make my own titlebar, similar to `ExtendsContentIntoTitleBar` and `SetTitleBar`.

**Comment by jeromelaban** [comment]
This is not supported at this time indeed. If GTK3 allows for it, then it may be possible to implement that feature.

**Reply by FrozDark** [reply]
What about the WPF? I'm pretty sure It allows it

**Reply by jeromelaban** [reply]
It's not supported by Skia+WPF either at this time, but if you're interested, we're acceptig contributions or we're offering professional support at info@platform.uno.

**Reply by ivanjx** [reply]
but is it possible to remove the default titlebar from gtk3 and drag the window my own?

**Reply by jeromelaban** [reply]
As I mentioned above, no it's not yet possible. Let's track in https://github.com/unoplatform/uno/issues/10368
